public/js/components/notification.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component, PropTypes } from 'react';
+import { gettext } from 'utils';
+
+import cx from 'classnames';
+
+
+export default class Notification extends Component {
+
+  static propTypes = {
+    autoHide: PropTypes.bool,
+    handleAutoHide: React.PropTypes.func,
+    handleDismissClick: React.PropTypes.func,
```

Comment:
I think you could replace these two props with `removeItem` and just call it like `this.removeItem()` for each case.